packages/yoastseo/spec/fullTextTests/testTexts/index.js

```diff
@@ -1,176 +1,103 @@
-// English papers
```

Comment:
I decided to rearrange the list of papers in alphabetical order, following the structure of the language folders. However, in almost every other file, the sequence follows the order the languages were added to yoastseo, so this rearrangement seems to be unnecessary, shrug emoji.

Reply:
It does look nicer this way. We can still follow this order when adding new language paper in the future, at least for full text test file :)